foundation-extensions/mobile-core/mobile-core-api-reference.md

```diff
@@ -131,11 +129,11 @@ If you installed and configured the Analytics extension, this method sends an An
 **Syntax**
 
 ```java
-public static void trackAction(final String action, final Map<String, String> contextData)
++ (void) trackAction: (nullable NSString*) action data: (nullable NSDictionary*) contextData;
```

Comment:
this update is incorrect.  the previous version is correct for the java signature.